source/gameengine/Ketsji/KX_2DFilterManager.cpp

```diff
@@ -101,8 +101,9 @@ KX_PYMETHODDEF_DOC(KX_2DFilterManager, addFilter, " addFilter(index, type, fragm
 	int index = 0;
 	int type = 0;
 	const char *frag = "";
+	int mipmap = 0;
 
-	if (!PyArg_ParseTuple(args, "ii|s:addFilter", &index, &type, &frag)) {
+	if (!PyArg_ParseTuple(args, "ii|s:addFilter", &index, &type, &frag, &mipmap)) {
```

Comment:
I doubt for this part. The user have the possibility to set the mipmap just after he created the filter as he sets uniforms.